dashboard/src/pages/Capability/Workloads/index.tsx

```diff
@@ -41,6 +41,15 @@ export default (): React.ReactNode => {
             title: {
               dataIndex: 'name',
             },
+            description: {
+              render: (text, row) => {
+                return (
+                  <Paragraph ellipsis={{ rows: 1, expandable: true, symbol: 'more' }}>
```

Comment:
add 'key' fix 'Each child in a list should have a unique "key" prop.'

Reply:
When will this issue will happen?

Reply:
open browser console.  
react keys doc: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
I went through the doc but I still cloud not figure out how to fix it. Can you make another PR afterwards? Thanks.

Reply:
This problem is not related to this PR and I will fix it